bigbone/src/main/kotlin/social/bigbone/api/method/admin/AdminIpBlocksMethods.kt

```diff
@@ -0,0 +1,112 @@
+package social.bigbone.api.method.admin
+
+import social.bigbone.MastodonClient
+import social.bigbone.MastodonRequest
+import social.bigbone.Parameters
+import social.bigbone.api.Pageable
+import social.bigbone.api.Range
+import social.bigbone.api.entity.admin.AdminIpBlock
+
+/**
+ * Allow management of ip addresses blocked and to be blocked.
+ * @see <a href="https://docs.joinmastodon.org/methods/admin/ip_blocks/">Mastodon admin/ip_blocks API methods</a>
+ */
+class AdminIpBlocksMethods(private val client: MastodonClient) {
+
+    private val adminIpBlocksEndpoint = "/api/v1/admin/ip_blocks"
+    private val ipSubdomainRegex = """^\\d+\\.\\d+\\.\\d+\\.\\d+/\\d+$""".toRegex()
```

Comment:
While I generally like the idea of this client-side check, I think this version doesn’t work:

1. It doesn’t say anywhere on the documentation page if these are IPv4 only or also IPv6 addresses. This regex would only match IPv4 addresses as valid
2. The regex is between `"""`, so we must not use double escapes (`\\`) like we need for `""`
3. (With the previous one fixed:) This regex would also allow for `999999999.999999999.999999999.1/12356679423687426348`. It’s too unspecific for client-side validation

I would suggest to remove it and not check the IP to be blocked locally.